src/uwtools/drivers/jedi.py

```diff
@@ -117,11 +117,9 @@ def validate_only(self):
         executable = file(Path(self._driver_config["execution"]["executable"]))
         config = self.configuration_file()
         yield [executable, config]
-        cmd = "%s && time %s --validate-only %s 2>&1" % (
-            " && ".join(self._driver_config["execution"]["envcmds"]),
-            refs(executable),
-            refs(config),
-        )
+        cmd = "time %s --validate-only %s 2>&1" % (refs(executable), refs(config))
+        if envcmds := self._driver_config["execution"].get("envcmds"):
+            cmd = " && ".join([*envcmds, cmd])
```

Comment:
`envcmds` is optional, but we were treating it as if it were required. This makes it optional again, prepending `cmd` with a `&&`-connected pipeline of all defined commands.